.huskyrc

```diff
@@ -0,0 +1,5 @@
+# This loads nvm.sh and sets the correct PATH before running hook
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+
+nvm use 14
```

Comment:
I don't think that it's a good idea to add it for all.

Reply:
it's better to add it inside your `~/.bashrc`

Reply:
Discussed.
Removed it and created a documentation